cdc/log.cc

```diff
@@ -845,21 +863,27 @@ class transformer final {
         std::transform(pc.begin(), pc.end(), std::back_inserter(columns), [](auto& c) { return &c; });
         std::transform(cc.begin(), cc.end(), std::back_inserter(columns), [](auto& c) { return &c; });
 
-        query::column_id_vector static_columns, regular_columns;
+        query::column_id_vector static_column_ids, regular_column_ids;
```

Comment:
Please don't rename variables like this when you change logic. If you really want to change the name do it afterward/beforehand.